docs/csharp/programming-guide/classes-and-structs/snippets/fields/Program.cs

```diff
@@ -3,25 +3,25 @@
 //<Snippet1>
 public class CalendarEntry
 {
-    // private field
-    private DateTime date;
-
     // public field (Generally not recommended.)
-    public string day;
+    public string Day;
+    
+    // private field
+    private DateTime _Date;
```

Comment:
The convention is to use `_date`.

Reply:
Agreed, this is not a standard naming convention.